model_external_socket/java/src/org/openda/externalsocket/SocketServer.java

```diff
@@ -0,0 +1,43 @@
+package org.openda.externalsocket;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+class SocketServer {
+
+	private int portNumber;
+
+	SocketServer(int portNumber) {
+		this.portNumber = portNumber;
+	}
+
+    void runAndWaitForMessage() {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=6157987)